src/ode_solver/ode_solver_base.cpp

```diff
@@ -16,7 +17,9 @@ ODESolverBase<dim,real,MeshType>::ODESolverBase(std::shared_ptr< DGBase<dim, rea
         , mpi_communicator(MPI_COMM_WORLD)
         , mpi_rank(dealii::Utilities::MPI::this_mpi_process(MPI_COMM_WORLD))
         , pcout(std::cout, mpi_rank==0)
-        {}
+{
+    limiter = BoundPreservingLimiterFactory<dim,real>::create_limiter(all_parameters, this->dg->nstate);
```

Comment:
call this after `: dg(dg_input)`; you can refer to the `FlowSolver` constructor for an example of this

Reply:
Included this after `: dg(dg_input)`.